app/api/helpers/files.py

```diff
@@ -191,7 +190,7 @@ def make_frontend_url(path, parameters=None):
     ))
 
 
-def create_save_pdf(pdf_data, dir_path='/static/uploads/pdf/temp/'):
+def create_save_pdf(pdf_data, key, dir_path='/static/uploads/pdf/temp/'):
```

Comment:
I would suggest doing a quick check on usage of this method in other places to ensure that you haven't broken anything else.

Reply:
Yes, did that :)